index.html

```diff
@@ -2433,21 +2455,55 @@ <h2>
           <ol class="algorithm">
             <li>Set the [=assigned launch consumer=] to |consumer|.
             </li>
-            <li>If the [=unconsumed launch params=] on the {{Window/window}} is
-            set:
+            <li>[=list/For each=] [=unconsumed launch params=]:
+            </li>
+            <li>
+              <ol>
+          <li>Call |consumer| with [=unconsumed launch params=].
+          </li>
+              </ol>
+            </li>
+            <li>Clear the list of [=unconsumed launch params=].
+            </li>
+          </ol>
+        </section>
+      </section>
+      <section>
+        <h3>
+          Handling Web App Launches
+        </h3>
+        <p>
+          The steps to <dfn>launch a web app</dfn> are given by the 
```

Comment:
The "launch a web app" process is unused. The "Execute a file handler launch" process should call into to this algorithm with its custom LaunchParams.

Reply:
Evan has a PR (#2) which does that.